py/circuitpy_mpconfig.h

```diff
@@ -837,6 +844,7 @@ extern const struct _mp_obj_module_t wifi_module;
     USTACK_MODULE \
     WATCHDOG_MODULE \
     WIFI_MODULE \
+    MSGPACK_MODULE \
```

Comment:
Alphabetical into the list please. (That way it's easy to find)

Reply:
Done